docs/snippets/react/my-component-play-function-composition.js.mdx

```diff
@@ -0,0 +1,36 @@
+```js
+// MyComponent.stories.js | MyComponent.stories.jsx | MyComponent.stories.ts | MyComponent.stories.tsx
+
+// These are placeholders until the addon-interaction is out
+
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { MyComponent } from './MyComponent.js';
+
+export default {
+  component: MyComponent,
+};
+
+export const FirstStory = {
+  play: async () => {
+    userEvent.type(screen.getByTestId('an-element'), 'example-value');
+  },
+};
+
+export const SecondStory = {
+  play: async () => {
+    await userEvent.type(screen.getByTestId('other-element'), 'another value');
+  },
+};
+
+export const CombinedStories = {
+  ...FirstStory,
```

Comment:
Is it necessary to spread FirstStory into CombinedStories? It doesn't seem like it would be, which could cause confusion by doing so.

Reply:
@kylegach this is one of those cases, technically you shouldn't, but what could happen is the following, a reader checks the docs, and the first question that he/she asks is, I can do this, but can I also combine them by spreading the story object ? With this we're indirectly telling the reader that he/she can do it.